src/View/Window.vala

```diff
@@ -425,17 +425,17 @@ public class Files.View.Window : Hdy.ApplicationWindow {
         var action_close_end = new SimpleAction ("tabmenu-close-end", null);
         var action_close_others = new SimpleAction ("tabmenu-close-others", null);
         var action_duplicate = new SimpleAction ("tabmenu-duplicate", null);
-        var action_new_window = new SimpleAction ("tabmenu-new-window", null);
+        var action_move_to_new_window = new SimpleAction ("tabmenu-new-window", null);
 
         add_action (action_close);
         add_action (action_close_end);
         add_action (action_close_others);
         add_action (action_duplicate);
-        add_action (action_new_window);
+        add_action (action_move_to_new_window);
 
         marlin_app.set_accels_for_action ("win.tabmenu-close", {"<Ctrl>W"});
         marlin_app.set_accels_for_action ("win.tabmenu-duplicate", {"<Ctrl><Alt>T"});
-        marlin_app.set_accels_for_action ("win.tabmenu-new-window", {"<Ctrl><Alt>N"});
+        marlin_app.set_accels_for_action ("win.tabmenu-move-to-window", {"<Ctrl><Alt>N"});
```

Comment:
This is only for the accel, so you have to change the action name below on line 444 as well